src/actions/index.js

```diff
@@ -0,0 +1,8 @@
+// export const CREATE_BOOK = ({ book }, state) => [
+//   ...state,
+//   book,
+// ];
```

Comment:
- [ ] Please remove this commented code before merging the branch

Reply:
Done!
Thank you Sergio!